pkg/sentry/seccheck/sinks/remote/remote.go

```diff
@@ -74,14 +91,20 @@ func setupSink(config map[string]any) (*os.File, error) {
 	if !ok {
 		return nil, fmt.Errorf("endpoint %q is not a string", addrOpaque)
 	}
-	return setup(addr)
+	file, _, err := setupWithVersion(addr)
+	return file, err
+}
+
+// setupWithVersion returns the file and the remote version.
+func setupWithVersion(path string) (*os.File, uint32, error) {
+	return setup(path)
```

Comment:
Empty implementation. I guess this is where you would want to process the remote's version and disable batching if the remote doesn't support it. Please add unit test to check the fallback behavior.